desktop-widgets/diveplanner.cpp

```diff
@@ -503,7 +503,7 @@ PlannerSettingsWidget::PlannerSettingsWidget(QWidget *parent, Qt::WindowFlags f)
 	connect(ui.bottompo2, SIGNAL(valueChanged(double)), plannerShared::instance(), SLOT(set_bottompo2(double)));
 	connect(ui.decopo2, SIGNAL(valueChanged(double)), plannerShared::instance(), SLOT(set_decopo2(double)));
 	connect(ui.bestmixEND, SIGNAL(valueChanged(int)), plannerShared::instance(), SLOT(set_bestmixend(int)));
-	connect(ui.bottomSAC, SIGNAL(valueChanged(double)), plannerShared::instance(), SLOT(sec_bottomsac(double)));
+	connect(ui.bottomSAC, SIGNAL(valueChanged(double)), plannerShared::instance(), SLOT(set_bottomsac(double)));
```

Comment:
again, that's because this is all the old syntax

@atdotde -- not sure if you have time right now... but at some point we should migrate this to the new `connect()` syntax that does compile time argument checks and avoids these issues.

Reply:
I will submit a bugfix2, that makes these changes, and also replace plannerModel with plannerShared in the connects. It will sit on top of this one, to make life easier for me.